commands/pipeline.go

```diff
@@ -0,0 +1,128 @@
+package commands
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/logrusorgru/aurora"
+)
+
+//use struct instead of interface
+//(json.Unmarshal causing big integer which pipline ID value to change)
+type pipline struct {
+	ID        int64  `json:"id"`
+	Sha       string `json:"sha"`
+	Ref       string `json:"ref"`
+	Status    string `json:"status"`
+	CreatedAt string `json:"created_at"`
+	UpdatedAt string `json:"updated_at"`
+	WebURL    string `json:"web_url"`
+}
+
+func displayMultiplePiplines(m []pipline) {
+	// initialize tabwriter
+	w := new(tabwriter.Writer)
+
+	// minwidth, tabwidth, padding, padchar, flags
+	w.Init(os.Stdout, 8, 8, 0, '\t', 0)
+
+	defer w.Flush()
+	if len(m) > 0 {
+		fmt.Printf("Showing piplines %d of %d on %s\n\n", len(m), len(m), GetEnv("GITLAB_REPO"))
+		for i := 0; i < len(m); i++ {
+
+			duration := TimeAgo(m[i].CreatedAt)
+
+			if m[i].Status == "success" {
+				_, _ = fmt.Fprintln(w, aurora.Green(fmt.Sprint("#", m[i].ID)), "\t", m[i].Ref, "\t", aurora.Magenta(duration))
+			} else {
+				_, _ = fmt.Fprintln(w, aurora.Red(fmt.Sprint("#", m[i].ID)), "\t", m[i].Ref, "\t", aurora.Magenta(duration))
+			}
+		}
+	} else {
+		fmt.Println("No Pipelines available on " + GetEnv("GITLAB_REPO"))
+	}
+}
+func deletePipeline(cmdArgs map[string]string, arrFlags map[int]string) {
+	issueID := strings.Trim(arrFlags[1], " ")
+	if CommandArgExists(cmdArgs, issueID) {
+		arrIds := strings.Split(strings.Trim(issueID, "[] "), ",")
```

Comment:
To avoid conflict, change the `issueID` to `pipelineID`....

Reply:
ok! great